modules/ROOT/pages/index.adoc

```diff
@@ -29,6 +29,8 @@ Hybrid::
 Your on-premises Mule servers host your app deployments and you use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
 +
 For applications that you deploy on-premises, you can send data to third-party analytics applications.
++
+If you are using MuleSoft Government Cloud, you can deploy Mule runtime engine to a private cloud or on-premises environment and manage your app deployments using your MuleSoft Government Cloud control plane. Refer to xref:gov-cloud::gov-cloud-standalone.adoc[Standalone Mule Support] for MuleSoft Government Cloud requirements.
```

Comment:
do we need to explicitly say app deployments? I think manage your deployments is clear enough.
xref:gov-cloud::gov-cloud-standalone.adoc[standalone MuleSoft support]? Lowercase since the xref is inline and not in See Also. Also, Mule or MuleSoft?

Reply:
Looks like it's a reference to the topic itself so title case is fine.

Reply:
Agree Valkyrie! Thanks!